src/ventures-for-shared-prosperity/what-we-provide.njk

```diff
@@ -7,14 +7,102 @@ title: What the ideas42 Venture Studio can provide
 {% block masthead %}
   {{ super() }}
   <div class="hero standard-restricted-width">
-    <div class="g-max-3 g-right">
+   {%include "components/cohort-banner.njk"%}
```

Comment:
I'm not sure where in the page, but looks like we have one or more elements that aren't closed properly. You can see it with View Source and by running it through the validator: https://validator.w3.org/nu/?doc=https%3A%2F%2Fdeploy-preview-29--ideas42ventures.netlify.app%2Fventures-for-shared-prosperity%2Fwhat-we-provide%2F

Reply:
@tylergaw Do you know what the difference is between the deployed version from that link and what I have locally? I ran it through the validator before committing and these errors aren't showing.

Reply:
@will0101 What are you running through the validator? Are you using the Direct Input?

Reply:
Fixed!